### PR TITLE
Enable contract nonces externalization

### PR DESCRIPTION
**Pull request by MiroslavGatsanoga**
**Description**:

**Related issue(s)**:

Fixes https://github.com/hashgraph/hedera-services/issues/7424

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by Nana-EC — APPROVED** [review]

File: hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/context/properties/BootstrapPropertiesTest.java [hunk 1]
```diff
@@ -564,6 +556,12 @@ class BootstrapPropertiesTest {
             entry(CONFIG_VERSION, 10),
             entry(RECORDS_USE_CONSOLIDATED_FCQ, false));
 
+    @LoggingTarget
```
Comment (david-bakin-sl) [inline hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/context/properties/BootstrapPropertiesTest.java:559]: Not sure why this got moved.  It was perfectly fine where it was.  But ok.

**Review by david-bakin-sl — APPROVED** [review]
LGTM

File: hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/context/properties/BootstrapPropertiesTest.java (hunk 1 above)
Comment (MiroslavGatsanoga) [reply hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/context/properties/BootstrapPropertiesTest.java:559]: Very strange, added it to [the follow up task](https://github.com/hashgraph/hedera-services/issues/7449) that will address the code style issues.

**Review by stoqnkpL — APPROVED** [review]

**Review by netopyr — APPROVED** [review]
LGTM in regards to changes in hedera-app

**Pull request merged**